google/cloud/exceptions.py

```diff
@@ -41,7 +43,10 @@ def __init__(self, message, errors=()):
         self._errors = errors
 
     def __str__(self):
-        return '%d %s' % (self.code, self.message)
+        result = u'%d %s' % (self.code, self.message)
+        if six.PY2:  # pragma: NO COVER  Python2
```

Comment:
For some reason I wasn't thinking it was that smart and it just had to run all the lines.